client/src/components/FourOhFour/index.js

```diff
@@ -4,7 +4,7 @@ import Spinner from 'react-spinkit';
 import { Link } from 'gatsby';
 
 import notFoundLogo from '../../assets/images/freeCodeCamp-404.svg';
-import { quotes } from '../../resources/quotes.json';
+import words from '../../../../config/motivationalQuotes.json';
```

Comment:
```suggestion
import words from '../../../../config/motivational-quotes.json';
```
This is a more standard filename convention.